src/style.scss

```diff
@@ -156,6 +156,10 @@ ul {
   padding: 1px 8px;
 }
```

Comment:
- [ ] Please you should record a video presenting the features of the project, the maximum length of the video is 5 minutes, the video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear and All the members in the video should appear in the video and talk about the project. don't forget. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.